source/termination_criteria/interface.cc

```diff
@@ -90,13 +88,11 @@ namespace aspect
       // call the execute() functions of all plugins we have
       // here in turns.
       std::list<std::string>::const_iterator  itn = termination_obj_names.begin();
-      for (typename std::list<std::unique_ptr<Interface<dim> > >::const_iterator
-           p = termination_objects.begin();
-           p != termination_objects.end(); ++p,++itn)
```

Comment:
here is another bug.

Reply:
Thanks. You don't have to find these bugs for me, though I very much appreciate that you do :-)